docs/mozillavpn.json

```diff
@@ -1,10 +1,10 @@
 {
   "latest": {
-    "ANDROID": "2.10.1",
-    "IOS": "2.10.0",
-    "LINUX": "2.10.1",
-    "MACOS": "2.10.0",
-    "WINDOWS": "2.10.0"
+    "ANDROID": "2.11.0",
```

Comment:
do we want to do the same for mozillavpn_staging.json too? Plus usually android and ios takes a bit more days to go live.

Reply:
Our home rolled system doesn't have the ability to set different dates for different releases. This file only has functional use for mac and windows. For other platforms it's just documentation.

Reply:
This is not true. iOS and Android uses the `/versions` guardian endpoint (settings -> about -> check for update). And exposing this change means that we show to mobile users that they can update to 2.11 when actually 2.11 does not exist for them (yet).
I would suggest reverting this change and exclude mobile until we release the client for real on mobile.